### PR TITLE
Remove deprecated components

### LINKED ISSUE #2223: Remove deprecated code
Remove all code marked for deprecation for version 2.0

### PR DESCRIPTION
**Pull request by deadlight**
## Done

I've removed all code marked as deprecated for 2.0.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- See that there are no visual changes to components

## Details

Fixes #2223

### CODE REVIEW

**Comment by kwm14** [comment]
## Design review
Percy serving one visual diff for components will be worth checking in the codebase as it could just be a Percy glitch.

**Before**
Screenshot 2019-03-19 at 12 19 39

**After**
Screenshot 2019-03-19 at 12 19 56

**Comment by deadlight** [comment]
@kwm14 no, Percy is correct. The example is using deprecated float utilities. I'll fix it!

**Comment by kwm14** [comment]
Nice. Thanks 👍

File: docs/en/base/code.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ If you want to refer to a larger piece of code, use <code>&lt;pre&gt;</code> tog
 View example of the base code block
 </a>
 
-### Copyable
+### Snippet
```
Comment (Caleb-Ellis) [inline docs/en/base/code.md]: Shouldn't this stay as Copyable?

Comment (deadlight) [reply docs/en/base/code.md]: Oh yes, my mistake. (I initially thought that we were keeping the other one)

**Pull request merged**